pay-api/src/pay_api/models/eft_credit.py

```diff
@@ -80,5 +80,6 @@ def get_eft_credits(cls, short_name_id: int) -> List[Self]:
         return (cls.query
                 .filter(EFTCredit.remaining_amount > 0)
                 .filter(EFTCredit.short_name_id == short_name_id)
+                .with_for_update()
```

Comment:
https://dev.to/ivankwongtszfung/safe-update-operation-in-postgresql-using-sqlalchemy-3ela#:~:text=Solution%202%3A%20Update%20Lock&text=In%20SQLAlchemy%2C%20there%20is%20a,this%20transaction%20release%20the%20lock.